src/index.html

```diff
@@ -1,17 +1,190 @@
 <!doctype html>
-<html lang="en">
+  <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="POST"
+        onsubmit="onSubmit()">
```

Comment:
```suggestion
    <form
        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
        method="POST"
        onsubmit="onSubmit()"
      >
```
Pay attention to fromatting. Closing bracket should be on the new line